src/main/java/ca/corefacility/bioinformatics/irida/constraints/ProjectRoleHasCorrectMetadataRole.java

```diff
@@ -13,51 +14,50 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * Validation annotation to validate that if the projectRole is set to PROJECT_OWNER
- * for a user/usergroup then the metadataRole must be set to the highest level
- * An array of fields (projectRole, metadataRole) must be supplied
- *
+ * Validation annotation to validate that if the projectRole is set to PROJECT_OWNER for a user/usergroup then the
+ * metadataRole must be set to the highest level An array of fields (projectRole, metadataRole) must be supplied
+ * <p>
  * Example, compare a single user/usergroup projectRole to it's metadataRole:
- * @ProjectRoleHasCorrectMetadataRole(first = "projectRole", second = "metadataRole", message = "server.project.owner.incorrect.metadata.role"))
- *
+ * {@literal @}ProjectRoleHasCorrectMetadataRole(first = "projectRole", second = "metadataRole", message =
+ * "server.project.owner.incorrect.metadata.role"))
  */
-@Target({TYPE, ANNOTATION_TYPE})
+@Target({ TYPE, ANNOTATION_TYPE })
 @Retention(RUNTIME)
 @Constraint(validatedBy = ProjectMetadataRoleValidator.class)
 @Documented
 public @interface ProjectRoleHasCorrectMetadataRole {
```

Comment:
Can we rename this to `MetadataRoleValidate`

Reply:
Renamed in 3c3136c4c1ab2e318d6a50d5a9639281bbdbb16a